src/examples/java/io/nats/examples/benchmark/NatsBench.java

```diff
@@ -265,7 +284,7 @@ void adjustAndSleep(Connection nc) throws InterruptedException {
             
             // Flush small messages regularly
             if (this.size < 64 && count != 0 && count % 100_000 == 0) {
-                try {nc.flush(Duration.ofSeconds(5));}catch(Exception e){}
+                try {nc.flush(Duration.ofSeconds(15));}catch(Exception e){}
```

Comment:
We still need this now?

Reply:
i don't know if we need the adjust but i was having issues with the flush on a slow network so I bumped it up

Reply:
I mean the flush for small messages inline in the benchmark. Client should do the right thing now yes?

Reply:
this was more for the pub/sub asymmetry. When I first wrote the lib i kept getting slow consumers. That might have gone away but I am tempted to leave this just in case, especially for windows and slower machines where i have seen bigger asymmetry issues.

Reply:
this one was not used, the one in throttled benchmark is used now, but added a comment.